app/src/main/java/com/dessalines/thumbkey/ui/components/keyboard/KeyboardScreen.kt

```diff
@@ -139,6 +141,8 @@ fun KeyboardScreen(
         val controllerKeys = listOf(EMOJI_BACK_KEY_ITEM, NUMERIC_KEY_ITEM, BACKSPACE_KEY_ITEM, RETURN_KEY_ITEM)
         val keyboardHeight = Dp((keySize * controllerKeys.size) - (keyPadding * 2))
 
+        ctx.currentInputConnection.requestCursorUpdates(0, 0)
```

Comment:
This line is also getting an IDE error about versions. And you shouldn't use 0, 0, but the imported modes.

Reply:
According to the documentation, you're supposed to pass in 0 to disable the updates (https://developer.android.com/reference/android/view/inputmethod/InputConnection#requestCursorUpdates(int,%20int)). I put that there to disable them for the emoji keyboard

Reply:
Is this the version error you see? 
`Call requires API level 33 (current min is 21): android.view.inputmethod.InputConnection#requestCursorUpdates`
Looks like the 2-parameter version of that function was added in 33, and I should use the 1-parameter version added in 21

Reply:
Yep.